contracts/dapis/DapiServer.sol

```diff
@@ -842,69 +844,22 @@ contract DapiServer is
         return dapiNameHashToDataFeedId[keccak256(abi.encodePacked(dapiName))];
     }
 
-    /// @notice Reads the data feed with ID
-    /// @param dataFeedId Data feed ID
+    /// @notice Reads the data feed with dAPI name hash
+    /// @param dapiNameHash dAPI name hash
     /// @return value Data feed value
     /// @return timestamp Data feed timestamp
-    function readDataFeedWithId(bytes32 dataFeedId)
+    function readDataFeedWithDapiNameHash(bytes32 dapiNameHash)
```

Comment:
Actually, I just saw it's being used in the proxy `DapiProxy` contract 🤔 maybe this function would make more sense to be in the caller contract since it seems it will be the only place where it is going to be called from.

Reply:
This function is meant to save on the number of external calls for dAPI reads. We don't want the proxy contract to have to `call()` DapiServer twice, once to get the data feed ID from the dAPI name (hash) and once more to read the data feed.